draft-ietf-quic-tls.md

```diff
@@ -613,16 +613,27 @@ by the "initial_max_data" transport parameter supplied by the server.  A client
 MUST treat receipt of a NewSessionTicket that contains an "early_data" extension
 with any other value as a connection error of type PROTOCOL_VIOLATION.
 
+A client that wishes to send early data uses the "early_data" extension in the
+ClientHello message of a subsequent handshake (see Section 4.10.2 of
+{{!TLS13}}). It then sends the application data in 0-RTT packets.
+
 Early data within the TLS connection MUST NOT be used.  As it is for other TLS
 application data, a server MUST treat receiving early data on the TLS connection
```

Comment:
Maybe irrelevant to this PR, but s/TLS connection/CRYPTO stream/.

Reply:
I'm not sure what would be the best way to phrase it. We have 3 different CRYPTO streams though, one for every encryption level.

Reply:
Good point. Rereading the sentence, I think we can remove it as a whole.

As of -12, we transmit TLS handshake messages instead of TLS records. That means that there is no way of transmitting 0-RTT "records" using the CRYPTO streams.

Reply:
Can we do that in a separate PR? I think it's orthogonal, and this PR already has a lot of commits.

Reply:
Makes sense. Filed PR #2845.